lib/checks/color/color-contrast.json

```diff
@@ -12,6 +12,7 @@
         "imgNode": "Element's background color could not be determined because element contains an image node",
         "bgOverlap": "Element's background color could not be determined because it is overlapped by another element",
         "fgAlpha" : "Element's foreground color could not be determined because of alpha transparency",
+        "elmOutsideParent": "Element's background color could not be determined because it extends beyond a parent node",
         "default": "Unable to determine contrast ratio"
```

Comment:
The default line comes up any time the check returns undefined but there isn't incompleteData passed through for it. The `elmOutsideParent` code path definitely caused it. I didn't see any others, but if we keep seeing it after this is merged we can identify and float up more reasons.